messagebus/src/vespa/messagebus/routing/routingnode.cpp

```diff
@@ -67,10 +67,8 @@ RoutingNode::~RoutingNode()
 void
 RoutingNode::clearChildren()
 {
-    for (std::vector<RoutingNode*>::iterator it = _children.begin();
-         it != _children.end(); ++it)
-    {
-        delete *it;
+    for (auto child : _children) {
```

Comment:
Could consider `auto*` here to make it more immediately obvious it's a raw pointer (also goes for other instances of for-each iteration over pointer containers)

Reply:
Done